modules/module_loader/include/ModulesApp.h

```diff
@@ -21,7 +100,131 @@ class ModulesApp : public MooseApp
 
   static void registerApps();
   static void registerAll(Factory & f, ActionFactory & af, Syntax & s);
+  template <typename T>
+  static void registerAllObjects(Factory & f, ActionFactory & af, Syntax & s);
   static void registerObjects(Factory & factory);
   static void associateSyntax(Syntax & syntax, ActionFactory & action_factory);
   static void registerExecFlags(Factory & factory);
 };
+
+///@{
+/**
+ * Dummy methods to clear unused parameter warnings from compiler
+ */
+void
+clearUnusedWarnings(Factory & /*factory*/)
+{
+}
+void
+clearUnusedWarnings(Syntax & /*syntax*/, ActionFactory & /*action_factory*/)
+{
+}
+///@}
+
```

Comment:
```suggestion
```

Reply:
I don't understand why these are here. I didn't put them in. They are called in the current registration methods. They've always been empty. Maybe something Robert was doing?